.github/workflows/pipeline.yml

```
@@ -0,0 +1,149 @@
+          # Our publish script creates the GitHub release (with the plugin zip
+          # attached); the action's own release-creation step would 409 on the
+          # tag we already published.
+          createGithubReleases: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PLUGIN_DB_TOKEN: ${{ secrets.PLUGIN_DB_TOKEN }}
+          # Optional: only needed if this repo has been renamed since being
+          # added to PluginDatabase as a submodule (URL-based lookup will miss).
+          PLUGIN_DB_SUBMODULE_PATH: plugins/non-steam-playtimes
+
+      - run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:63: Double quote to prevent globbing and word splitting

(shellcheck)